### ISSUE TITLE
Brazilian format is wrong

### ISSUE THREAD

**Issue by CavalcanteLeo** [NikKovIos/NKVPhonePicker#21]
We have 2 patterns:

` +55 (XX) XXXXX-XXXX `
and
` +55 (XX) XXXX-XXXX `

**Comment by NikKovIos** [comment]
Imagine the user type +55 than any digits. How the app should be available to distinguish between two patterns? Are there any characteristics which inherent to one and another pattern? Maybe unique digit sequence of concrete numbers?

**Comment by CavalcanteLeo** [comment]
this might be a good help

https://gist.github.com/MauricioMoraes/2885d4975ac5e52d0ad7

**Comment by CavalcanteLeo** [comment]
yep, it changes the mask when the last character is typed, also it change where the hyphen "-" is

I think i've done a long time ago a mask for this case using objective-c as well, i will try to find it tomorrow. Dunno if i still have the code.

**Comment by NikKovIos** [comment]
Thanks🙆, ill watch this closely in a free time.
